Support/Multitenancy/Multiple-Ironic-conductors/handle-images.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+N_NODES=${N_NODES:-1000}
+IMAGE_NAMES=(
+  "quay.io/metal3-io/ironic-ipa-downloader"
+  "quay.io/metal3-io/ironic:v26.0.1"
+  "quay.io/metal3-io/ironic-client"
+  "quay.io/metal3-io/keepalived:v0.2.0"
+  "quay.io/metal3-io/mariadb:latest"
+  "gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0"
+  "quay.io/metal3-io/metal3-fkas:latest"
+)
+
+for image in "${IMAGE_NAMES[@]}"; do
+  if [[ $(docker images --format "{{.Repository}}:{{.Tag}}" | grep "${image}") == "" ]]; then
```

Comment:
```suggestion
  if ! docker images --format "{{.Repository}}:{{.Tag}}" | grep -q "${image}"; then
```
Use return values rather than output.

Besides, if this is just a check if you should be pulling or not, just pull. Its lightweight check inside docker.